app/assets/stylesheets/application/components/like-button.scss

```diff
@@ -1,17 +1,22 @@
 .like-button {
-  span {
+  > .wrapper {
     color: $color-gray-dark-light;
+    font-size: 14px;
+  }
 
-    &:hover {
-      cursor: pointer;
-    }
+  > .wrapper:hover {
+    cursor: pointer;
+  }
 
-    &.liked {
-      color: $brand-primary;
-    }
+  > .wrapper.liked {
+    color: $brand-primary;
+  }
+
+  > .wrapper > .icon {
```

Comment:
Merge rule `> .wrapper > .icon` with rule on line 2<br>Selector should have depth of applicability no greater than 2, but was 3